lists/ye/ye-mopic.json

```
@@ -0,0 +1,50 @@
+{
+  "name": {
+    "en": "Yemen Ministry of Planning and International Cooperation",
+    "local": "وزارة التخطيط والتعاون الدولي "
+  },
+  "url": "https://mopic-yemen.org/",
+  "description": {
+    "en": "The Ministry of Planning and International Cooperation registers foreign non-profit and non-governmental organisations with a physical location and address in Yemen.\n\nOrganisations are granted a registration number that is provided on a certificate granting or renewing the license. \n\nThis registration is renewed every two years but the number does not change. Registered organisations must re-register upon changing their name, logo, or nature of their activities. "
+  },
+  "coverage": [
+    "YE"
+  ],
+  "subnationalCoverage": [],
+  "structure": [
+    "charity",
+    "company",
+    "trust"
+  ],
+  "sector": [],
+  "code": "YE-MOPIC",
+  "confirmed": false,
+  "deprecated": false,
+  "listType": "secondary",
+  "access": {
+    "availableOnline": false,
+    "onlineAccessDetails": null,
+    "publicDatabase": "",
+    "guidanceOnLocatingIds": "Ask the organisation to provide their 'Principal Cooperation Agreement'. The identifier will be provided as the Registration Number.",
+    "exampleIdentifiers": "YE-MOPIC-0036",
```

```suggestion
    "exampleIdentifiers": "0036",
```
You don't need to include the prefix.